src/shapes/IText/IText.ts

```diff
@@ -39,7 +39,7 @@ export const iTextDefaultValues = {
   caching: true,
   hiddenTextareaContainer: null,
   _selectionDirection: null,
-  _reSpace: /\s|\n/,
+  _reSpace: /\s|\r?\n/,
```

Comment:
Since you are not so old, do you know what \r is and why is there together with \n ?

Reply:
I don't...
Return carriage?
Something from the past?

Reply:
I don't care too much about it but maybe for compatibility

Reply:
it was the special code that made the printer head go back to start of the line, while the '\n' was advancing the paper o ne line.
If you didn't add `\r` you would print on the same line of paper over and over.

Or sometimes you would do

```
BOLD TEXT\n
BOLD TEXT\r\n
```
to print twice on purpose and have the print darker.

things i learned working with old dot matrix printers

Reply:
interesting
so \r should be added, right?

Reply:
> interesting ~so \r should be added, right?~

so \r should NOT be added, right?